protocol/message.go

```diff
@@ -118,6 +201,12 @@ func NewKeyLookupInEpochProof(ap *m.AuthenticationPath,
 	}, e
 }
 
+// NewMonitoringProof creates the response message a CONIKS server sends
+// to a client upon a MonitoringRequest. The contained DirectoryProofs
+// includes a list of proofs of inclusion and a list
+// of STRs for the epoch range [startEpoch, endEpoch], where startEpoch
+// and endEpoch are the epoch range endpoints indicated in the client's
+// request. The error code is set to Success.
```

Comment:
"The error code is set to Success." <- That's confusing. But also in combination of what the constructor is doing: One can pass it an `ErrorCode` as a parameter (can be `Success` or anything else). This `ErrorCode` `e` is incorporated into the freshly created Response (OK) *and* returned (why)?